### PR TITLE
Update Developer Guide with Enhanced `Add` command and add high importance flag feature

### PR DESCRIPTION
**Pull request by xSaints19x**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -286,7 +334,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | long-time user                            | separate past and current clients                                                    | avoid contacting clients with similar names or old clients that I am not presently working with   |
 | `* *`    | new user                                  | view a list of commands                                                              | know what commands are available and the right commands to use                                    |
 | `*`      | interior designer                         | add images under a client                                                            | keep track of images such as floor plans and inspirational designs relevant to the client         |
-| `*`      | interior designer                         | add a high importance tag to a client                                                | take note of pressing issues regarding a client, such as mobility issues                          |
```
Comment (takufunkai) [inline docs/DeveloperGuide.md:289]: Since we're adding this, it might be better to up the priority of the user story!

**Review by takufunkai — CHANGES_REQUESTED** [review]
LGTM! Might just need a small tweak and we're good to go.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (xSaints19x) [reply docs/DeveloperGuide.md:289]: Thanks for the reminder! 👍 I'll push the changes asap

**Review by takufunkai — APPROVED** [review]
LGTM!

**Pull request merged**